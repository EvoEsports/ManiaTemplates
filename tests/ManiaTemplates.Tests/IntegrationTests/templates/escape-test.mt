<component>
    <property type="List<int>" name="data"/>

    <template>
        <label foreach="int i in data" if="i > 1 && i < 4" text="{{ i }}" data-cond="{{ i >= 0 }}"/>
        <label foreach="int i in data" if="i < 1 &amp;&amp; i == i" text="{{ i }}" data-cond='{{ i > i }}'/>
    </template>
</component>