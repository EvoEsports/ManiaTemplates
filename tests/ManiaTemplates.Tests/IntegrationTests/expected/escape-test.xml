<manialink version="3" id="MtEscapeTest" name="EvoSC#-MtEscapeTest">
<label text="2" data-cond="True" />
<label text="3" data-cond="True" />
<label text="0" data-cond="False" />
</manialink>
